integrated_channels/api/v1/logs/serializers.py

```diff
@@ -85,7 +112,14 @@ class GenericLearnerSyncStatusSerializer(LearnerSyncStatusSerializer):
 
     class Meta:
         model = GenericLearnerDataTransmissionAudit
-        fields = '__all__'
+        fields = (
+            'user_email',
+            'content_title',
+            'progress_status',
+            'sync_status',
+            'sync_last_attempted_at',
+            'friendly_status_message',
+        )
```

Comment:
nit: Unless I'm missing something, it seems we're using the same fields for all the LMS providers.  Can we break this list out into a shared constant?

Reply:
we can add these fields to the base `LearnerDataTransmissionAudit` serializer!